model/src/main/java/com/ge/predix/acs/model/Obligation.java

```diff
@@ -0,0 +1,97 @@
+package com.ge.predix.acs.model;
```

Comment:
Do not forget to add/(run the script) the License for newly created files

Reply:
done